src/components/ImageSelector/carouselAnimation.css

```diff
@@ -0,0 +1,461 @@
+    );
+  }
+  85% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0.1) 91%
+    );
+  }
+  90% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0.3) 94%
+    );
+  }
+  95% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0.5) 97%
+    );
+  }
+  100% {
+    opacity: 0.8;
+  }
+}
+
+.unzoomTransitionImageFastEnd {
+  animation: unMaskTransitionImageFastEnd 1.5s linear 0s normal forwards;
+}
+
+@keyframes unMaskTransitionImageFastEnd {
+  0% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 40%
+    );
+  }
+  5% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 43%
+    );
+  }
+  10% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 46%
+    );
+  }
+  15% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 49%
+    );
+  }
+  20% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 52%
+    );
+  }
+  25% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 55%
+    );
+  }
+  30% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 58%
+    );
+  }
+  35% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 61%
+    );
+  }
+  40% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 64%
+    );
+  }
+  45% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 67%
+    );
+  }
+  50% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 70%
+    );
+  }
+  55% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 73%
+    );
+  }
+  60% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 76%
+    );
+  }
+  65% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 79%
+    );
+  }
+  70% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 82%
+    );
+  }
+  75% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 85%
+    );
+  }
+  80% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0) 88%
+    );
+  }
+  85% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
+      rgba(0, 0, 0, 0.1) 91%
+    );
+  }
+  90% {
+    mask-image: radial-gradient(
+      circle at 50% 50%,
+      black 10%,
```

Comment:
can you check to not specifiy again all the value that do not change (I guess it works), would make it much lighter to read and understand the animations

Reply:
Unfortunately, seems that not works. radial gradient function needs the informations to compute the gradient. I search and try but that doesn't work.